src/hiero_sdk_python/query/query.py

```
@@ -91,6 +92,40 @@ def set_query_payment(self, payment_amount: Hbar) -> "Query":
         """
         self.payment_amount = payment_amount
         return self
+    
+    def set_max_query_payment(self, max_query_payment: Union[int, float, Decimal, Hbar]) -> "Query":
```

Ruff (0.14.10)

Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)